cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -154,300 +130,3 @@ def store_flow_cell_data(self, parsed_flow_cell: FlowCellDirectoryData) -> None:
-            self.finish_flow_cell(
-                bcl_converter=bcl_converter,
-                flow_cell_name=flow_cell_dir.name,
-                flow_cell_path=flow_cell_dir,
-            )
-
-
-class DemuxPostProcessingNovaseqAPI(DemuxPostProcessingAPI):
-    """Post demultiplexing API class for Novaseq flow cell."""
-
-    def rename_files(self, demux_results: DemuxResults) -> None:
-        """Rename the files according to how we want to have it after demultiplexing is ready."""
-        LOG.info(f"Renaming files for flow cell {demux_results.flow_cell.full_name}")
-        flow_cell_id: str = demux_results.flow_cell.id
-        for project_dir in demux_results.raw_projects:
-            files.rename_project_directory(
-                project_directory=project_dir, flow_cell_id=flow_cell_id, dry_run=self.dry_run
-            )
-
-    def add_to_cgstats(self, demux_results: DemuxResults) -> None:
-        """Add the information from demultiplexing to cgstats"""
-        create.create_novaseq_flowcell(manager=self.stats_api, demux_results=demux_results)
-
-    def fetch_report_samples(self, flow_cell_id: str, project_name: str) -> List[StatsSample]:
-        samples: List[StatsSample] = self.stats_api.find_handler.project_sample_stats(
-            flowcell=flow_cell_id, project_name=project_name
-        )
-        LOG.info(
-            "Found samples: %s for flow cell: %s, project: %s",
-            ",".join(sample.sample_name for sample in samples),
-            flow_cell_id,
-            project_name,
-        )
-        return samples
-
-    @staticmethod
-    def sample_to_report_line(stats_sample: StatsSample, flow_cell_id: str) -> str:
-        lanes: str = ",".join(str(lane) for lane in stats_sample.lanes)
-        return "\t".join(
-            [
-                stats_sample.sample_name,
-                flow_cell_id,
-                lanes,
-                ",".join(str(unaligned.read_count) for unaligned in stats_sample.unaligned),
-                str(stats_sample.read_count_sum),
-                ",".join(str(unaligned.yield_mb) for unaligned in stats_sample.unaligned),
-                str(stats_sample.sum_yield),
-                ",".join(str(unaligned.q30_bases_pct) for unaligned in stats_sample.unaligned),
-                ",".join(str(unaligned.mean_quality_score) for unaligned in stats_sample.unaligned),
-            ]
-        )
-
-    @staticmethod
-    def get_report_lines(stats_samples: List[StatsSample], flow_cell_id: str) -> Iterable[str]:
-        """Convert stats samples to format lines ready to print."""
-        for stats_sample in sorted(stats_samples, key=lambda x: x.sample_name):
-            yield DemuxPostProcessingNovaseqAPI.sample_to_report_line(
-                stats_sample=stats_sample, flow_cell_id=flow_cell_id
-            )
-
-    @staticmethod
-    def write_report(report_path: Path, report_data: List[str]) -> None:
```

Comment:
This report I am unsure of whether it would be required - if so we need to implement functionality for this where we access data from cg